### PR TITLE
Conversion to Desktop version

### PR DESCRIPTION
**Pull request by Rifat8080**
- Added media query @768 px 
- Customized all elements according to size and colors for desktop version
- Customize submission form according to desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -239,6 +284,12 @@ <h1 class="foot-title">
                 required
               />
             </li>
+            <li class="input-full-name">
+                        <input type="text" name="first_name" id="first_name" maxlength="30" placeholder="First name"
+                            required>
+                        <input type="text" name="last_name" id="last_name" maxlength="30" placeholder="Last name"
+                            required>
+                    </li>
```
Comment (bobb-Rob) [inline index.html:292]: - [ ] The contact section on the desktop lacks left and right padding as seen in the Figma design. Please fix this.

#### Your design
image

#### Figma Design
image

File: index.html [hunk 2]
```diff
@@ -62,14 +89,22 @@ <h1>
...
       </div>
     </section>
 
     <!-- My recent Works Section -->
 
     <section>
-      <div class="jobs" id="jobs-id">
+      <div class="recent-job">
         <h1>My Recent Works</h1>
+      </div>
```
Comment (bobb-Rob) [inline index.html:105]: - [ ] The green bar is not visible in the desktop design as observed in the Figma Design. Kindly remove it

Your work
image

Figma Design
image

File: index.html [hunk 3]
```diff
@@ -9,16 +9,43 @@
...
+              </li>
+              <li class="menu-item">
+                <a href="#footer-container-id" class="link-contact">
+                  Contact
+                </a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
```
Comment (bobb-Rob) [inline index.html:39]: - [ ] Your desktop view looks very good but the mobile view does not look good any longer. Kindly ensure that the your app on mobile still has proper margin, padding, and element in the right position.

image

- [ ] Please go through all the sections and ensure that they look good on mobile view as well. I noticed that your contact section is scattered in mobile view.

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Rifat8080, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @Rifat8080 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- Good UI :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**